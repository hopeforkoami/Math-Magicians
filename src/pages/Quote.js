import React from 'react';

function Quote() {
  return (
    <div className="quote">
      <p className="quote_citation">
        Mathematics is not about numbers, equations, computations, or algorithms: it is about
        understanding. â€“William Paul Thurston.
      </p>
    </div>
  );
}

export default Quote;
