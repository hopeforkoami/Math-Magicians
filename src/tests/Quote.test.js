import { render } from '@testing-library/react';
import { BrowserRouter as Router,} from 'react-router-dom';
import Quote from '../pages/Quote';

test('render Quote component', () => {
  const { asFragment } = render(<Router><Quote /></Router>);
  expect(asFragment()).toMatchSnapshot();
});
