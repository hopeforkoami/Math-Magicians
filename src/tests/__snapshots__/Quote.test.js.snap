// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Quote component 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <header>
      <div
        class="project_title"
      >
        Math Magicians
      </div>
      <nav>
        <ul
          class="menu"
        >
          <a
            href="/"
          >
            <li
              class="menu_element start_menu_element selected_menu_element"
            >
              Home
            </li>
          </a>
          <a
            href="/calcul"
          >
            <li
              class="menu_element"
            >
              Calculator
            </li>
          </a>
          <a
            href="/quote"
          >
            <li
              class="menu_element"
            >
              Quote
            </li>
          </a>
        </ul>
      </nav>
    </header>
    <section
      class="section quote_section"
    >
      <div
        class="section_content"
      >
        <p
          class="quote_citation"
        >
          Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston.
        </p>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
